### ISSUE TITLE
(Self-Voicing) Keyboard Shortcuts dialog?

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#267]
Keyboard Shortcuts Dialog. What to do about this for Self-Voicing?

**Questions:**
1. Is it desirable to include KB Shortcuts dialog in SVing?
2. Is it feasible?
3. If feasible, would large reading blocks like the example work?
4. If large blocks would work, I propose the following for GFLB.

**Example content for highlighted block “Move Spheres” block (same as PDOM):**
Move sphere left and right with Left and Right Arrow keys.
Move in smaller steps with Shift plus Left and Right Arrow keys.
Move in larger steps with Page Up and Page Down keys.
Jump to left with Home key.
Jump to right with End key.

**Sketch of dialog with large reading blocks:**
- Black squares just show the separate blocks.
- The yellow block shows active reading (mouse click, touch, focus, gesture focus)
- The black box with icon is the mouse hover with no auto-self-voicing.

![Screen Shot 2021-02-19 at 5 18 48 PM](https://user-images.githubusercontent.com/1902116/108562908-2b9c4d00-72db-11eb-9770-b4942d16633c.png)

**Comment by emily-phet** [comment]
This sounds great to me. It breaks up the text some, but not into lots of small chunks. Let's give this a try.

**Comment by terracoda** [comment]
@jessegreenberg, you'll see that I intentionally did not include the heading for the blocks on the left because the heading is included in the first line of the instructions.

For the group of Basic Actions, that is not the case, so that's why the heading is included there. So, for the "Basic Actions" block, you need to add a period after the heading, like this:

Basic Actions.
Move to next item or group with Tab key.
Move to previous item or group with Shift plus Tab key.
Move between items in a group with Left and Right arrow keys or Up and Down arrow keys.
Press buttons with Space key.
Toggle checkboxes with Space key.
Exit a dialog with Escape key.

**Comment by terracoda** [comment]
I am not sure how it would sound with out a period, like this:

Basic Actions
Move to next item or group with Tab key.
Move to previous item or group with Shift plus Tab key.
Move between items in a group with Left and Right arrow keys or Up and Down arrow keys.
Press buttons with Space key.
Toggle checkboxes with Space key.
Exit a dialog with Escape key.

We could try that first to see how it sounds. Also, I am not sure it there are line breaks or not, nor do I know how web speech would handle line breaks in terms of pause-length.

If you know any of these details, that would be helpful for me.

**Comment by terracoda** [comment]
Actually, thinking more long term, a colon might be better, like this:

Basic Actions:
Move to next item or group with Tab key.
Move to previous item or group with Shift plus Tab key.
Move between items in a group with Left and Right arrow keys or Up and Down arrow keys.
Press buttons with Space key.
Toggle checkboxes with Space key.
Exit a dialog with Escape key.

**Comment by terracoda** [comment]
Assigning to you @jessegreenberg, and note that I created a similar issue for John Travoltage as that sim will be published first, see https://github.com/phetsims/john-travoltage/issues/400

If you need anything for either issue let me know.

**Comment by jessegreenberg** [comment]
This was done in the above commits and can be heard at https://phet-dev.colorado.edu/html/gravity-force-lab-basics/1.1.0-dev.35/phet/gravity-force-lab-basics_en_phet.html if you would like to have a listen @terracoda.

I went ahead with this one because it is related to cleanup I wanted to do for john-travoltage.

**Comment by terracoda** [comment]
This is great @jessegreenberg, but I noticed that X-close button, does voice. I'll check John Travoltage, and open a new issue for that.
